ApplicationCode/ProjectDataModel/RimWellPathGroup.cpp

```diff
@@ -19,11 +19,13 @@
 
 #include "RiaTextStringTools.h"
 #include "RigWellPath.h"
+#include "RimModeledWellPathLateral.h"
 
 #include "cafPdmFieldScriptingCapability.h"
 #include "cafPdmObjectScriptingCapability.h"
 #include "cafPdmUiTreeOrdering.h"
 
+#include <QRegularExpression>
```

Comment:
Can this be removed? Looks like QRegExp is used instead of QRegularExpression

Reply:
done